api/v1/controllers/collectors.js

```diff
@@ -116,8 +116,9 @@ function postCollector(req, res, next) {
      * When a collector registers itself with Refocus, Refocus sends back a
      * special token for that collector to use for all further communication
      */
-    o.dataValues.token = jwtUtil
-      .createToken(collectorToPost.name, collectorToPost.name);
+    o.dataValues.token = jwtUtil.createToken(
+      collectorToPost.name, collectorToPost.name, { IsCollector: true }
```

Comment:
I think it should include all the other details, too (IsAdmin=false, IsBot=false, etc.)